examples/loader-google-bigquery/README.md

```diff
@@ -0,0 +1,7 @@
+[Framework examples →](../)
```

Comment:
this file needs to be updated

Reply:
Something like below could work? Just need to test that the "view live" when final changes is merged :)

# Google BigQuery Data Loader

View live: <https://observablehq.observablehq.cloud/framework-example-loader-google-bigquery/>

This Observable Framework example demonstrates how to write a JavaScript data loader that fetches metrics from the Google Bigquery API  using the [Google BigQuery Data Node.js Client](https://cloud.google.com/nodejs/docs/reference/bigquery-connection/latest). The data loader fetches the `confirmed_cases` metric (based on a publicly available Covid-19 BigQuery dataset) for a specified date range and then displays the data in a line chart. The data loader lives in [`src/data/covidstats_it.csv.js`](./src/data/covidstats_it.csv.js) and uses the helper [`src/data/google-bigquery.js`](./src/data/google-bigquery.js).

Reply:
looks good!

Reply:
Will you directly do all suggested trims/changes and merge after (if possible)? I don't mind, let me know what works best for you / what will be the fastest for you 🙏

Reply:
yes I can take it from here